src/lib/server/websearch/scrape/types.ts

```diff
@@ -0,0 +1,5 @@
+export type SerializedHTMLElement = {
```

Comment:
nit. stylisitically, in this case, we have been using `interface` over `type`

Reply:
I think this should be `interface` & was left from https://github.com/huggingface/chat-ui/pull/1094/commits/492c51fff022ba40410056c0889b974f14a83471